azure-pipelines.yml

```diff
@@ -1,28 +1,77 @@
-# Android
-# Build your Android project with Gradle.
-# Add steps that test, sign, and distribute the APK, save build artifacts, and more:
-# https://docs.microsoft.com/azure/devops/pipelines/languages/android
-
+variables:
+  sdk: 'iphoneos'
+  configuration: 'Build'
+  msMapsModulesPath: 'iOS/MSMapsModules'
+  msMapsModulesToRootPath: '../../'
+  msMapsModulesWorkspacePath: '$(msMapsModulesPath)/MSMapsModules.xcworkspace'
+  testAppKeysPath: '$(msMapsModulesPath)/MSMapsModulesTestApp/keys.plist'
+  referenceAppPath: 'iOS/MSMapsModulesReferenceApp'
+  referenceAppWorkspacePath: '$(referenceAppPath)/MSMapsModulesReferenceApp.xcworkspace'
+  
 trigger:
 - master
 
-pool:
-  vmImage: 'windows-latest'
-
-steps:
-
-  # Create dummy secrets.gradle
-- bash: |
-    echo ext.credentialsKey = "\"DummyKey\"" > secrets.gradle
-    echo `<secrets.gradle`
-  displayName: Create secrets.gradle
-  workingDirectory: 'Android/Samples/app'
+jobs:
+- job: Build_Android
+  pool:
+    vmImage: 'windows-latest'
+  steps:
+    # Create dummy secrets.gradle
+  - bash: |
+      echo ext.credentialsKey = "\"DummyKey\"" > secrets.gradle
+      echo `<secrets.gradle`
+    displayName: Create secrets.gradle
+    workingDirectory: 'Android/Samples/app'
+  - task: Gradle@2
+    inputs:
+      workingDirectory: 'Android/Samples'
+      gradleWrapperFile: 'Android/Samples/gradlew'
+      gradleOptions: '-Xmx3072m'
+      publishJUnitResults: false
+      testResultsFiles: '**/TEST-*.xml'
+      tasks: 'build'
 
-- task: Gradle@2
-  inputs:
-    workingDirectory: 'Android/Samples'
-    gradleWrapperFile: 'Android/Samples/gradlew'
-    gradleOptions: '-Xmx3072m'
-    publishJUnitResults: false
-    testResultsFiles: '**/TEST-*.xml'
-    tasks: 'build'
+- job: Build_iOS
+  pool:
+    vmImage: 'macOS-10.14'
+  steps:
+  - bash: |
+      cd $(msMapsModulesPath)
+      pod install
+      cd $(msMapsModulesToRootPath)
+    displayName: 'pod install for MSMapsModules'
+  - bash: |
+      echo '<?xml version="1.0" encoding="UTF-8"?> <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd"> <plist version="1.0"> <dict> <key>credentialsKey</key> <string>DummyKey</string> </dict> </plist>' > $(testAppKeysPath)
+    displayName: 'Create keys.plist for MSMapsModulesTestApp'
+  - bash: |
+      cd $(referenceAppPath)
+      pod install
+      cd $(msMapsModulesToRootPath)
+    displayName: 'pod install for MSMapsModulesReferenceApp'
+  - bash: |
+      echo '<?xml version="1.0" encoding="UTF-8"?> <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd"> <plist version="1.0"> <dict> <key>credentialsKey</key> <string>DummyKey</string> </dict> </plist>' > $(referenceAppPath)/keys.plist
+    displayName: 'Create keys.plist for MSMapsModulesReferenceApp'
+  - task: Xcode@5
```

Comment:
Can you add displayName for each of the xcode tasks so it's clearer what it's doing?